2025-10-02 22:49:11.626629: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:467] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1759438151.642887 4180475 cuda_dnn.cc:8579] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1759438151.648163 4180475 cuda_blas.cc:1407] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
W0000 00:00:1759438151.662309 4180475 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
W0000 00:00:1759438151.662356 4180475 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
W0000 00:00:1759438151.662362 4180475 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
W0000 00:00:1759438151.662366 4180475 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/cluster/work/sebasrb/zbot-policy-walking/train.py", line 14, in <module>
    import ksim
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/__init__.py", line 17, in <module>
    from .task import *
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/__init__.py", line 3, in <module>
    from .amp import *
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/amp.py", line 33, in <module>
    from ksim.task.ppo import PPOConfig, PPOTask, PPOVariables
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/ppo.py", line 22, in <module>
    from ksim.task.rl import RLConfig, RLLoopCarry, RLLoopConstants, RLTask
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/rl.py", line 644, in <module>
    class RLTask(xax.Task[Config], Generic[Config], ABC):
                 ~~~~~~~~^^^^^^^^
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/typing.py", line 379, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/typing.py", line 1864, in __class_getitem__
    _check_generic(cls, params, len(cls.__parameters__))
  File "/cluster/home/sebasrb/.local/lib/python3.11/site-packages/typing_extensions.py", line 3136, in _check_generic
    raise TypeError(f"Too {'many' if alen > elen else 'few'} arguments"
TypeError: Too few arguments for <class 'xax.task.task.Task'>; actual 1, expected at least 2
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/cluster/work/sebasrb/zbot-policy-walking/train.py", line 14, in <module>
    import ksim
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/__init__.py", line 17, in <module>
    from .task import *
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/__init__.py", line 3, in <module>
    from .amp import *
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/amp.py", line 33, in <module>
    from ksim.task.ppo import PPOConfig, PPOTask, PPOVariables
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/ppo.py", line 22, in <module>
    from ksim.task.rl import RLConfig, RLLoopCarry, RLLoopConstants, RLTask
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/site-packages/ksim/task/rl.py", line 644, in <module>
    class RLTask(xax.Task[Config], Generic[Config], ABC):
                 ~~~~~~~~^^^^^^^^
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/typing.py", line 379, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/cluster/work/sebasrb/zbot-policy-walking/ksim-env/lib/python3.11/typing.py", line 1864, in __class_getitem__
    _check_generic(cls, params, len(cls.__parameters__))
  File "/cluster/home/sebasrb/.local/lib/python3.11/site-packages/typing_extensions.py", line 3136, in _check_generic
    raise TypeError(f"Too {'many' if alen > elen else 'few'} arguments"
TypeError: Too few arguments for <class 'xax.task.task.Task'>; actual 1, expected at least 2
